sunpy/sun/models.py

```diff
@@ -2,26 +2,27 @@
 Solar Physical Models
 ---------------------
 This module contains standard models of the sun from various sources. All data is saved
-in pandas DataFrames with two added attributes
+in Astropy QTables with an added attribute
 
 * source : names the source of the data
-* units : a dictionary with the units of each of the columns
 
 Object
 ------
-    interior : pandas.DataFrame
+    **interior** : ``astropy.table.QTable``
```

Comment:
```suggestion
    interior : `astropy.table.QTable`
```

Reply:
I didn't understand this properly before, my bad. Will make the changes.